evgMrmApp/src/evgAcTrig.h

```diff
@@ -22,8 +29,8 @@ class evgAcTrig : public mrf::ObjectInst<evgAcTrig> {
     void setBypass(bool);
     bool getBypass() const;
 
-    void setSyncSrc(bool);
-    bool getSyncSrc() const;
+    void setSyncSrc(epicsUInt16);
+    epicsUInt16 getSyncSrc() const;
```

Comment:
I think the c++ implicit conversion rules are such that existing code expecting `bool` will continue to work since `0` and `1` will continue to have the same meanings as `false` and `true`.